scripts/parse_file_list_for_projects.sh

```
@@ -0,0 +1,119 @@
+
+if [ "$GIT_CMD" == "" ]; then
+  GIT_CMD=$(which git)
+fi
+
+if [ "$PROJECT_ROOT" == "" ] || [ "$PROJECT_ROOT" == "." ]; then
+  repository_root=$($GIT_CMD rev-parse --show-toplevel)
+  PROJECT_ROOT=$repository_root
+  USE_ROOT=true
+fi
+
+SCRIPT_DIR=$(cd $(dirname $0); pwd)
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)